IPlug/IPlugConstants.h

```diff
@@ -29,7 +34,10 @@ enum EIPlugKeyCodes
 
 //TODO: these should be in a namespace, to avoid conflicts with third-party libraries
 static const double PI = 3.141592653589793238;
+
+// Magic number. Approximates 20 * log10(x) for gain to dB conversion
```

Comment:
Reverse engineered this, see http://www.wolframalpha.com/input/?i=plot+8.685889638065036553+*+log(x),+20+*+log10(x)

Should help for anyone wondering what it's for!

Reply:
like